MANUAL.md

```diff
@@ -0,0 +1,113 @@
+## System configuration
+
+1. Install the NixOS as it's shown in the [blog post](#).
+
+2. Install the cachix package.
+
+```
+nix-env -iA cachix -f https://cachix.org/api/v1/install
+```
+
+3. Add the 3mdeb binary cache.
+
+```
+cachix use 3mdeb
+```
+
+3. Install the git package.
+
+```
+nix-env -iA nixos.git
+```
+
+4. Clone the 3mdeb nixpkgs fork.
+
+```
+git clone https://github.com/3mdeb/nixpkgs.git -b landing-zone
```

Comment:
@Asiderr There is no such branch anymore. Do you mean this MR: https://github.com/3mdeb/nixpkgs/pull/1 ? I can merge it  and let's use master here. using branch will quickly become obsolete.

Reply:
@macpijan @Asiderr LZ should have tags and release process, tags should be not removed from tree ever.

Reply:
I'm not sure if it has and it's not only LZ here, but multiple of packages. But yes, using tag for the blog post content would be optimal for reproducibility in the future.